ground/src/main/java/com/google/android/ground/repository/SubmissionRepository.kt

```diff
@@ -172,8 +174,8 @@ constructor(
     )
 
   private fun applyAndEnqueue(mutation: SubmissionMutation): @Cold Completable =
-    localDataStore
-      .applyAndEnqueue(mutation)
+    submissionStore
+      .commitThenEnqueue(mutation)
```

Comment:
Why the change from apply --> commit?

Also, because of the new files, git/GitHub aren't able to highlight which methods were renamed. Let's move the rename into its own PR or its own commit to make it clear?

Reply:
Ah, sorry about that -- yeah I can revert the rename and separate it out. The motivation was that kotlin has a stdlib scope function called `apply` (https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/apply.html) so using `apply` in kotlin code requires ugly backticking.